### DISCUSSION TITLE
Cannot find ManageReportSchedule in Reports API

### DISCUSSION THREAD

**Discussion by yingchihpeng** [python-amazon-mws/python-amazon-mws#213] [General]
Hi,

My name is Ying and I'm new to this library, I'm trying to use 'ManageReportSchedule' as stated in MWS documentation to get order reports:
https://docs.developer.amazonservices.com/en_US/reports/Reports_ManageReportSchedule.html

But I don't seem to find a function that does that after going through the source code of mwc.py, can anyone guide me to the right direction? Much appreciated!

**Comment by GriceTurrble** [comment]
Hi Ying,

That particular call is missing at the moment. You can see relevant code in `develop` branch from `mws/apis/reports.py` is commented out, as it has not been completed yet.

We have a large set of changes being worked on currently in #207 . Once that's through, we can accept a change that adds this call to our `develop` branch, or you can make use of the generic request feature being added by that PR.

Will keep this open for tracking until we have that request method added. Thank you!

**Reply by yingchihpeng** [reply]
Thanks for the info :)
